R/standardize.models.R

```diff
@@ -123,9 +123,19 @@ standardize.default <- function(x,
 
   resp <- NULL
   if (!include_response) {
-    resp <- unique(c(insight::find_response(x), insight::find_response(x, combine = FALSE)))
+    resp <- unique(
+      c(
+        insight::clean_names(insight::find_response(x)),
+        insight::clean_names(insight::find_response(x, combine = FALSE))
+      )
+    )
   } else if (include_response && two_sd) {
-    resp <- unique(c(insight::find_response(x), insight::find_response(x, combine = FALSE)))
```

Comment:
@strengejacke these expressions are identical in the `if` and in the `else`, is this expected?

Reply:
Hm, doesn't look right. @mattansb you also worked on this function, any ideas?

Reply:
I'll have to take a closer look in a few days... But basically any change to this file will break everything :(

Reply:
Cleaned up the code to prevent duplicate lines

Reply:
Thanks